src/style.css

```diff
@@ -1,3 +1,83 @@
+:root{
+   --add-blue-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform:uppercase;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  padding: 0 50px;
+  display:flex;
+  justify-content:space-between;
+  align-items: center;
+  background-color: #fff;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logoImg {
+  height: 100%;
+  width:100%;
+}
+
+.nav {
+  display:block;
+}
+
+.nav__list {
+  display:flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+
+.link {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  text-decoration:none;
+  white-space: nowrap;
+  color:#000;
+  height:60px;
+  margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children
Also you can use `:not(:first-child)` to set a margin for all items besides first one